orm.py

```diff
@@ -145,6 +153,31 @@ def _schema(self):
             resp = await make_a_querry("""SELECT * FROM {} WHERE {}={}""".format(cls._name, field, value))
         return [cls(**dict(r)) for r in resp]
 
+    @classmethod
+    async def get_by_join(cls, *args, **kwargs):
+        allowed_fields = [c.name for c in cls._schema]
+
+        tables = "{}, {}".format(cls._name, ", ".join(args))
+        query = """SELECT {} FROM {} WHERE""".format(
+            ", ".join(allowed_fields),
+            tables
+        )
+
+        for i, kw in enumerate(kwargs):
+            if isinstance(kwargs[kw], (dict, list)):
+                kwargs[kw] = json.dumps(kwargs[kw])
+            if isinstance(kwargs[kw], str):
+                query += " {}='{}'".format(kw, kwargs[kw])
+            else:
+                query += " {}={}".format(kw, kwargs[kw])
+            if i + 1 < len(kwargs):
+                query += " AND "
+
+        print(query)
```

Comment:
print ;)

Reply:
True, debugging gone wrong; I'll remove that in a sec ;)